src/main/java/io/github/lucasstarsz/fastj/math/Maths.java

```diff
@@ -136,4 +136,17 @@ public static float magnitude(Point p) {
     public static boolean floatEquals(float a, float b) {
         return Math.abs(a - b) < FloatPrecision;
     }
+
+    /**
+     * Returns the value of lerping between x and y, by the interpolation point t.
+     *
+     * @param x The first {@code double} value.
+     * @param y THe second {@code double} value.
+     * @return The lerped value
```

Comment:
Use proper punctuation for your `@return` statement -- add a period to the end.

Reply:
You'll also want to make sure to update your documentation here -- it's no longer a double 😉

> Use proper punctuation for your `@return` statement -- add a period to the end.

Don't forget about this either!